public/data/Kendall_Does_Coding.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Abdus Salam",
+  "type": "personal",
+  "bio": "Hello. My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, Javascript, and the one I'm using to make this website, HTML & CSS.",
+  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "links": [
+    {
+      "name": "Check out my personal website",
+      "url": "https://kendalldoescoding.gq/",
+      "icon": "globe"
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/kendalldoescoding",
+      "icon": "github"
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "githlab"
```

Comment:
There's no GitLab icon. If you want GitLab to be added, open an issue [**here**](https://github.com/EddieHubCommunity/LinkFree/issues/choose), we discuss it, then we decide to add it or not. 🙂

Reply:
That's anyway the wrong spelling I put, I'll create a issue shortly.

Reply:
#1045